pkg/drainer/drainer.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2016-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package drainer
+
+import (
+	nthAppConfig "github.com/aws/aws-node-termination-handler/pkg/config"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/kubectl/pkg/drain"
+	"log"
```

Comment:
Typically, you want to have standard imports above 3rd party (and "local") imports. Running `go fmt && go imports` on your code would automatically rearrange these imports accordingly.

https://godoc.org/golang.org/x/tools/cmd/goimports

Cool to leave this as-is though and I can push a new PR that adds a `go fmt/imports` check test.

Reply:
ah yeah, I had switched my formatter in vs code because it was removing packages when I was breaking stuff out for the reorg.

Reply:
I'll break up this PR into better chunks :)